aggregator/aggregator.go

```diff
@@ -112,7 +117,7 @@ func NewMocked(fun, regex, prefix, sub, outFmt string, cache bool, interval, wai
 		Interval:     interval,
 		Wait:         wait,
 		DropRaw:      dropRaw,
-		aggregations: make(map[uint]map[string]Processor),
+		aggregations: make(map[uint]aggregation),
```

Comment:
not using a pointer here makes the code fragile.
Having to replace the map entry after incrementing the counter just seems overly complicated and will likely lead to future bugs.